assets/js/atomic/blocks/product-elements/price-v2/block.json

```diff
@@ -0,0 +1,49 @@
+{
+	"$schema": "http://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-price-v2",
+	"title": "Price",
+	"keywords": "WooCommerce",
+	"category": "woocommerce-product-elements",
+	"textdomain": "woo-gutenberg-products-block",
+	"description": "Display the price of a product, including any discounts.",
+	"supports": {
+		"layout": {
+			"allowSwitching": false,
+			"allowInheriting": false,
+			"default": { "type": "flex" }
+		},
+		"html": false
+
+	},
+	"usesContext": [ "queryId", "postId" ],
+	"providesContext": {
+		"woocommerce/isDescendentOfSingleProductTemplate":
+			"isDescendentOfSingleProductTemplate",
+		"woocommerce/isDescendentOfSingleProductBlock":
+			"isDescendentOfSingleProductBlock",
+		"woocommerce/withSuperScriptStyle": "withSuperScriptStyle"
```

Comment:
Not fully implemented yet, I added this as a context value so that it could be used to provide the ability for child blocks to keep their superscript style separate from the parent block. It's possible that isn't great UX in which case this property could potentially just be removed from block context.